src/settings/MailSettingsViewer.ts

```diff
@@ -81,7 +81,7 @@ export class MailSettingsViewer implements UpdatableSettingsViewer {
 		// normally we would maybe like to get it as an argument but these viewers are created in an odd way
 		locator.mailAddressTableModelForOwnMailbox().then((model) => {
 			this.mailAddressTableModel = model
-			m.redraw()
+			model.init().then(m.redraw)
```

Comment:
the model can't handle multiple `init()`'s in a correct way currently

Reply:
right, need to fix that

Reply:
My only concern is that it ends up in the bundle but otherwise sure